jenkins_cli/cli.py

```diff
@@ -71,6 +72,22 @@ def xml_to_string(root):
     return ElementTree.tostring(root, encoding=('unicode' if sys.version_info[0] == 3 else None))
 
 
+def check_nonnegative(value):
```

Comment:
Hello @radomirbosak thank you for PR!

Maybe it's better to move ```check_nonnegative``` to ```__init__.py``` (because we have all argument parsing logic there) or even in dedicated file(arg_validators.py).

What do you think?